.buildkite/pipeline.yml

```diff
@@ -135,6 +135,14 @@ steps:
           run: android-jvm
     command: 'bash ./scripts/run-cpp-check.sh'
 
+  - label: ':android: Lint mazerunner scenarios'
+    depends_on: "android-jvm"
```

Comment:
Will this need to change after the docker image rejig?

Reply:
Yes - this depends on which PR lands first.

Reply:
Switched target to the `docker-reorg` branch as that will land first.